docs/core/compatibility/globalization/8.0/twodigityearmax-default.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Breaking change: TwoDigitYearMax default is 2049"
+description: Learn about the globalization breaking change in .NET 8 where the default value of the GregorianCalendar.TwoDigitYearMax property has changed from 2029 to 2049.
+ms.date: 01/25/2023
+---
+# TwoDigitYearMax default is 2049
+
+Calendar classes, such as <xref:System.Globalization.GregorianCalendar>, have a `TwoDigitYearMax` property that defines the last year of a 100-year range that can be represented by a two-digit year. This property is often used to translate a two-digit year to a four-digit year. Previously, <xref:System.Globalization.GregorianCalendar.TwoDigitYearMax?displayProperty=nameWithType> defaulted to 2029. That value meant that two-digit years from 00 to 29 translated to 2000-2029. Two-digit years from 30 to 99 translated to 1930-1999. The default `TwoDigitYearMax` property value for <xref:System.Globalization.GregorianCalendar> has now changed from 2029 to 2049. The new value means that two-digit years from 00 to 49 are translated to 2000-2049. Any year from 50 to 99 will be translated to 1950-1999.
```

Comment:
Are there any other calendar classes this applies to?

Reply:
Yes it does. It does apply on calendars which are based on Gregorian calendar like EastAsianLunisolarCalendar. Also it apply to calendar similar to Gregorian like JulianCalendar